### PR TITLE
 Startup-Business-Confernce-Page

### PR DESCRIPTION
**Pull request by ShingiraiBhengesa**
**Loom Presentation**
https://www.loom.com/share/3cf09e781442418592656418f70bc2c8

**In this branch, I created the following pages**
- [ ] The home page.
- [ ] The about page

**I also implemented the following features**
- [ ] Created the speaker's section dynamically using javascript
- [ ] Mobile and desktop responsiveness
- [ ] Hamburger menu

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+                  </div>
+              </div>
+              <div class="main-card">
+                  <div class="main-card-img">
+                      <i class="fas fa-chalkboard-teacher"></i>
+                  </div>
+                  <div class="main-card-title">
+                      <p>
+                          TS Exhibition
+                      </p>
```
Comment (micheaol) [inline index.html]: Please ensure that the pages should look almost identical to the original design. Kindly implement the `hover effect` on the cards.

Your Implementation | Design
:---:|:---:
image | image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+      <button type="button" class="main-btn">
+          Join CC Global Summit 2015
+      </button>
+      <a href="https://google.com" class="main-link-btn">See the all program</a>
+  </div>
+</main>
+<div class="container speakers">
+  <div class="speaker-title">
+      <p>Featured Speakers</p>
+  </div>
```
Comment (micheaol) [inline index.html:197]: Please ensure that the pages should look almost identical to the original design. 

Your Implementation | Design
:---:|:---:
image | image

File: about.html [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+                </p>
+            </div>
+            <div class="past-event-card-2">
+                <h2 class="past-event-date">
+                    2011
+                </h2>
+                <p class="past-event-site">
+                    TS GLobal Summit 2011 in Warsaw
+                </p>
+            </div>
```
Comment (micheaol) [inline about.html:171]: Please ensure that the pages should look almost identical to the original design. Kindly include an image in this `section` as in the original design. Also, please implement the `overlay hover` effect as in the original design.
 

Your Implementation | Design
:---:|:---:
image | image

File: main.css [hunk 4]
```diff
@@ -0,0 +1,621 @@
...
+body {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+
+  --primary-color: #272a31;
+  --secondary-color: #d3d3d3;
+  --accent-color: #ec4542;
+  --shadow: 0 2px 10px -1px rgba(0, 0, 0, 0.3);
+
```
Comment (micheaol) [inline main.css:13]: Please ensure that the pages use the exact same colors, for text, backgrounds, borders, etc, from the original design.

* Basic color swatch: #272A31, #EC5242, #D3D3D3

Also, please ensure that all page elements have similar spacing, borders, shadows, and paddings as in the original design.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,234 @@
...
+      <div class="main-title">
+          <p>Main Program</p>
+      </div>
+      <div class="main-program-container">
+          <div class="main-cards">
+              <div class="main-card">
+                  <div class="main-card-img">
+                      <i class="fas fa-book-reader"></i>
+                  </div>
+                  <div class="main-card-title">
```
Comment (micheaol) [inline index.html]: Please ensure that the HTML uses semantic tags, sections use `section`, articles use `article`, the nav and footer use `nav` and `footer` according to the project requirement.

File: main.js [hunk 6]
```diff
@@ -0,0 +1,161 @@
...
+      </div>
+      <div class="speaker-card-description">
+          <p>${dataSpeakers[index].description}</p>
+      </div>
+  </div>
+  `;
+
+const deployTwoSpeakers = () => {
+  speakerContainerCard.innerHTML = ' ';
+  // eslint-disable-next-line no-plusplus
```
Comment (micheaol) [inline main.js:119]: Please ensure that `Linter` is not disabled. `Linter` is provided to help improve your code. Kindly enable the disabled `Linter`.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,234 @@
...
+						<ul class="header-nav-desktop">
+								<li><a href="./about.html">About</a></li>
+								<li><a href="#">Program</a></li>
+								<li><a href="#">Join</a></li>
+								<li><a href="#">Sponsor</a></li>
+								<li><a href="#">News</a></li>
+								<li><a href="#">CC Campaign</a></li>
+						</ul>
+				</div>
+		</nav>
```
Comment (micheaol) [inline index.html]: Please ensure that the pages should look almost identical to the original design. I understand that you use `border box` as the `hover effect` for your `nav link` but in the original design, the last `item` has a `border red box` around it. Kindly implement that according to the project requirement. 

Your Implementation | Figma Design
:---:|:---:
image | image

File: index.html [hunk 8]
```diff
@@ -0,0 +1,234 @@
...
+          Join CC Global Summit 2015
+      </button>
+      <a href="https://google.com" class="main-link-btn">See the all program</a>
+  </div>
+</main>
+<div class="container speakers">
+  <div class="speaker-title">
+      <p>Featured Speakers</p>
+  </div>
+  <div class="speaker-container-card">
```
Comment (micheaol) [inline index.html:198]: Please ensure that each of your `speaker cards` is well aligned as in the original design.
Your Implementation | Design
:---:|:---:
image | image

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @ShingiraiBhengesa , 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html [hunk 9]
```diff
@@ -0,0 +1,205 @@
...
+    <main class="container">
+        <h2 class="main-title">
+            Creative Commons Global Summit 2015 Logo
+        </h2>
+        <p class="main-paragraph">
+            The logo of CC Global Summit 2015 was decided through the logo competition from 8. June to 7. July
+        </p>
+        <p class="main-paragraph-desktop">
+            The logo of CC Global Summit 2015 was decided through the logo competition from 8. June to 7. July. After
+            online public voting and Selection Committee's judging process, the logo from Naresh Agrawal won the prize.
```
Comment (MeqdamAlqudah) [inline about.html:139]: - [ ] Please as the last reviewer use the `section` tag to identify sectioning content.

File: index.html [hunk 10]
```diff
@@ -34,7 +34,7 @@
 								<li><a href="#">Join</a></li>
 								<li><a href="#">Sponsor</a></li>
 								<li><a href="#">News</a></li>
-								<li><a href="#">CC Campaign</a></li>
+								<li><a href="#" class="campain">CC Campaign</a></li>
```
Comment (MeqdamAlqudah) [inline index.html]: - [ ] Please as the last reviewer said implement the hovering effects on the buttons 

| Your design | 
| ------------- |
| image |

Comment (MeqdamAlqudah) [inline index.html]: > Please as the last reviewer said ensure that each of your `speaker cards` is well aligned as in the original design.
> 
> Your Implementation	Design
> image	image

**Review by MeqdamAlqudah — CHANGES_REQUESTED** [review]
Hi @ShingiraiBhengesa , 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Suggested changes
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: index.html (hunk 10 above)
Comment (ShingiraiBhengesa) [reply index.html]: Hello @MeqdamAlqudah please kindly note that there are no buttons, that is a menu item with  border and I have implemented it just like the design, also there is a hover effect on my nav elements

Comment (ShingiraiBhengesa) [reply index.html]: Hello @MeqdamAlqudah, kindly note that I updated the speaker cards and the  screenshot you provided is not my current design

**Comment by mke2111** [comment]
Project Under Second Opinion Review.
Kindly wait for a response. 😄

**Review by mke2111 — COMMENTED** [review]
Hello @ShingiraiBhengesa  👋,

_This is an additional review as a result of the second opinion request._

Great work done so far with this capstone, well done 👍🏽 

> ISSUE ONE: The Hover effects on the `CC Campaign` tab

Consider this as an optional request since you have implemented this request.

> ISSUE TWO: Usage of semantic tags for the respective elements

As best stated in the [HTML and CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md), specifically the first point, it emphasizes the usage of HTML 5 semantic tags over divs. I strongly suggest that you apply these tags where necessary in both HTML files. Consider this as a mandatory request that will benefit your project in numerous ways, and you as a developer.

Make the requested changes and submit another CRR whenever you are ready.

Keep rocking 🚀!

File: index.html [hunk 11]
```diff
@@ -0,0 +1,234 @@
...
+      </div>
+      <button type="button" class="main-btn">
+          Join CC Global Summit 2015
+      </button>
+      <a href="https://google.com" class="main-link-btn">See the all program</a>
+  </div>
+</main>
+<div class="container speakers">
+  <div class="speaker-title">
+      <p>Featured Speakers</p>
```
Comment (mke2111) [inline index.html]: This would best be a `section` tag instead of a `div` tag.

File: index.html [hunk 12]
```diff
@@ -0,0 +1,234 @@
...
+				</div>
+		</div>
+</header>
+<main>
+  <div class="container">
+      <div class="main-title">
+          <p>Main Program</p>
+      </div>
+      <div class="main-program-container">
+          <div class="main-cards">
```
Comment (mke2111) [inline index.html]: This would also make a good section with a `section` tag instead of a `div` tag.

**Review by keroloussamy — APPROVED** [review]
Hi @ShingiraiBhengesa, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️
- Add all the requirements. ✔️

clap



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**